docs/src/main/asciidoc/spring-cloud-bus-pubsub.adoc

```diff
@@ -0,0 +1,44 @@
+== Spring Cloud Bus
+
+Using link:https://cloud.google.com/pubsub/[Cloud Pub/Sub] as the link:https://spring.io/projects/spring-cloud-bus[Spring Cloud Bus] implementation is as simple as importing the `spring-cloud-gcp-starter-bus-pubsub` starter.
+
+Maven coordinates, using <<getting-started.adoc#_bill_of_materials, Spring Cloud GCP BOM>>:
+
+[source,xml]
+----
+<dependency>
+  <groupId>org.springframework.cloud</groupId>
+  <artifactId>spring-cloud-gcp-starter-bus-pubsub</artifactId>
+</dependency>
+----
+
+Gradle coordinates:
+
+[source,groovy]
+----
+dependencies {
+    compile group: 'org.springframework.cloud', name: 'spring-cloud-gcp-starter-bus-pubsub'
+}
+----
+
+=== Configuration Management with Spring Cloud Config
+
+Spring Cloud Bus can be used to push configuration changes from Spring Cloud Config server to the clients listening on the same bus.
+
+To use GCP Pub/Sub as the bus implementation, both configuration server and configuration client need `spring-cloud-gcp-starter-bus-pubsub` dependency.
+
+All other configuration is standard to https://spring.io/projects/spring-cloud-config[Spring Cloud Config].
+
+image::images/spring_cloud_bus_over_pubsub.png[]
```

Comment:
Where you have the cloud with the question mark, presumably that's a human. Maybe show a user symbol instead?
Also, for (5), do you want to say that the client *pull* the configuration changes? Otherwise, it's not clear if it's pushed or pulled.

Reply:
It does not have to be a human -- it could be an automated configuration update. The question mark signifies that we neither know nor care who/what updated the configuration.

I'll change "retrieve" to "pull", that's definitely less ambiguous.

Reply:
Since I am already updating the diagram, I've changed the wording to "A new version of externally stored configuration becomes available." to make it less passive tense.

Reply:
... and ditching the cloud.